### PR TITLE
fix: reset gamut ButtonBase styles to default

### PR DESCRIPTION
**Pull request by hmafzal**
## Title

Resets the gamut button base styles to be default styling.

https://codecademy.atlassian.net/browse/WEB-590

image

---

## Merging your changes

When you are ready to merge to master, use the "squash and merge" button in GitHub, and follow the [commit message guide](/README.md#commit-message-guide) to write your commit message.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.8.32-alpha.735af8.0<br/>@codecademy/gamut@5.0.1-alpha.735af8.0<br/></pre>

**Comment by JoshuaKGoldberg** [comment]
omg I'm so hyped for this

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e99de2ddfea7001736c532c--codecademy-styleguide.netlify.app>https://5e99de2ddfea7001736c532c--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e99de2ddfea7001736c532c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.6-alpha.c2f9ee.0<br/>@codecademy/gamut@7.0.4-alpha.c2f9ee.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e9f7280b48880018f4b9435--codecademy-styleguide.netlify.app>https://5e9f7280b48880018f4b9435--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e9f7280b48880018f4b9435>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.36db17.0<br/>@codecademy/gamut@7.1.2-alpha.36db17.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea07da680ebc80284698cee--codecademy-styleguide.netlify.app>https://5ea07da680ebc80284698cee--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea07da680ebc80284698cee>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.9929b2.0<br/>@codecademy/gamut@7.1.2-alpha.9929b2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea0af24f044e8d49694465e--codecademy-styleguide.netlify.app>https://5ea0af24f044e8d49694465e--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea0af24f044e8d49694465e>Deploy Logs</a></p>

File: packages/gamut/src/index.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ export * from './AppBar/AppBarSection';
 export * from './AppBar/AppBarTab';
 export * from './Badge';
 export * from './Banner';
-export * from './Button';
 export * from './ButtonBase';
+export * from './Button';
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/index.tsx:9]: If this actually was causing issues, we should probably comment here so I don't auto-sort in VS Code for fun some day..

Comment (jakemhiller) [reply packages/gamut/src/index.tsx:9]: yeah, imports are not a case where we want to alphabetize things. i'll add a comment.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.0b43d9.0<br/>@codecademy/gamut@7.1.2-alpha.0b43d9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea0b6c067c433ea3731c63d--codecademy-styleguide.netlify.app>https://5ea0b6c067c433ea3731c63d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea0b6c067c433ea3731c63d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.6f6cbe.0<br/>@codecademy/gamut@7.1.2-alpha.6f6cbe.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea11c956f215616f3e6836c--codecademy-styleguide.netlify.app>https://5ea11c956f215616f3e6836c--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea11c956f215616f3e6836c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.dff6bc.0<br/>@codecademy/gamut@7.1.2-alpha.dff6bc.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea1234067c43365a131c735--codecademy-styleguide.netlify.app>https://5ea1234067c43365a131c735--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea1234067c43365a131c735>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.aa077b.0<br/>@codecademy/gamut@7.1.2-alpha.aa077b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea1271131a310d5dcdc9b01--codecademy-styleguide.netlify.app>https://5ea1271131a310d5dcdc9b01--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea1271131a310d5dcdc9b01>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.8-alpha.c70718.0<br/>@codecademy/gamut@7.1.2-alpha.c70718.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea1b4b6c791abd79308d336--codecademy-styleguide.netlify.app>https://5ea1b4b6c791abd79308d336--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea1b4b6c791abd79308d336>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.9-alpha.533177.0<br/>@codecademy/gamut@7.1.3-alpha.533177.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea1c0e296ccd002533b02ec--codecademy-styleguide.netlify.app>https://5ea1c0e296ccd002533b02ec--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea1c0e296ccd002533b02ec>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.9-alpha.fee518.0<br/>@codecademy/gamut@7.1.3-alpha.fee518.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.0.9-alpha.b6716b.0<br/>@codecademy/gamut@7.1.3-alpha.b6716b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ea1c433453ee87d5cb802a0--codecademy-styleguide.netlify.app>https://5ea1c433453ee87d5cb802a0--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ea1c433453ee87d5cb802a0>Deploy Logs</a></p>

**Pull request merged**